README-SNAP.md

```diff
@@ -251,30 +277,30 @@ you are using the default.
 ```bash
 /snap/bin/couchdb.remsh -n couchdb@localhost -c $COOKIE
 ```
+
 # Building this snap <a name="building"></a>
 
-This build requires `core20`. On Ubuntu 18.04, you will need to do the following:
+The snapcraft tool can be installed from the snap store as such
 
-```
-sudo apt remove snapcraft
-sudo snap remove snapcraft
+```bash
 sudo snap install snapcraft --classic
 ```
 
-Then building is easy:
-
-```bash
-$ snapcraft
-```
+If you run snapcraft on your base system it will start either a mutlipass or lxd
+container and execute the installation within there. 
 
-The self-built snap will need to be installed using `--dangerous`:
+This can be tedious if errors occur. An alternative is to create your own lxd 
+container and run snapcraft in destructive mode (within the LXD container).
 
 ```bash
-sudo snap install ./couchdb_3.0.0_amd64.snap --dangerous
+> lxc launch ubuntu-daily:22.04 cdb
+> lxc shell cdb
+$ snapcraft --destructive-mode --verbose
 ```
 
-Clean up with:
+Once the snap has been built, the snap can be installed locally using `--dangerous`:
```

Comment:
this `--dangerous` mode is only for testing local snaps? When installing from the official "Snap repos" you would install couchdb without this flag?

Reply:
Correct. --dangerous is only needed when installing a snap that you've rolled your own. And even then, I don't think the adjective is apt: the snap is still embedded within AppAmour. Snap that are uploaded to the store have an extra check to see that they aren't tainted by external links.